rust_wtx_bench/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM docker.io/library/debian:bookworm-slim AS build
```

Comment:
Let's use a regular Rust image to avoid manual work.

Reply:
Done